client/state/themes/actions.js

```diff
@@ -705,6 +713,7 @@ export function pollThemeTransferStatus( siteId, transferId, interval = 3000, ti
 					dispatch( transferStatus( siteId, transferId, status, message, uploaded_theme_slug ) );
 					if ( status === 'complete' ) {
 						// finished, stop polling
+						dispatch( recordTracksEvent( 'calypso_automated_transfer_complete', { transfer_id: transferId } ) );
```

Comment:
Yeah that one doesn't play so well with `withAnalytics`. I'm updating the test to expect 4 calls